terraform-ss-proxy/reload-apache.cgi

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Set variables
+S3_BUCKET="REPLACE_WITH_S3_BUCKET_NAME"
+ENVIRONMENT="REPLACE_WITH_ENVIRONMENT_NAME"
+LOCAL_FILE="/etc/apache2/sites-enabled/unity-cs.conf"
+TEMP_FILE="/tmp/unity-cs.conf"
+SLACK_WEBHOOK=$(aws ssm get-parameter --name "/unity/shared-services/slack/apache-config-webhook-url" --with-decryption --query "Parameter.Value" --output text)
+
+# Function to send message to Slack and exit
+send_to_slack() {
+    local message="$1"
+    local env_prefix="[Unity-venue-${ENVIRONMENT}] "
+    curl  --silent --output /dev/null -X POST -H 'Content-type: application/json' \
+        --data "{\"text\":\"${env_prefix}${message}\"}" \
+        "${SLACK_WEBHOOK}"
+}
+
+# Download files from S3
+aws s3 sync s3://${S3_BUCKET}/ /etc/apache2/venues.d/ --exclude "*" --include "*.conf" --delete --quiet
```

Comment:
For this, it doesn't seem like we have a nice failure state- that is, if the `configtest` **fails**, we still have a *broken* configuration in-place, right?

Could we have something to copy-out the original config, then pull the new config, and copyback on failure?
(something as simple as `rsync -ax --delete /etc/apache2/venues.d/ /etc/apache2/venues.d.reload_safe/` and vice versa)